src/ansys/units/quantity_dimensions.py

```diff
@@ -245,6 +245,7 @@ class QuantityDimensions:
     WALL_ADJACENT_TEMPERATURE = TEMPERATURE
     WALL_TEMPERATURE_THIN = TEMPERATURE
     WALL_SHEAR_STRESS = STRESS
+    VOLUME_FRACTION = Dimensions()
```

Comment:
@hpohekar 

The one last piece is to extend the catalog in te `variable_descriptors` modules itself. The clue how to do it is in the unit tests: `catalog.add("WALL_SHEAR_STRESS_2", QuantityDimensions.STRESS)`.
We simply need to add the two variables, mentioned in the previous comment above.

Reply:
@seanpearsonuk 

I'm able to access them as follows:

```
>>> from ansys.units import VariableCatalog              
>>> VariableCatalog.VOLUME_FRACTION_PRIMARY_PHASE.name
'volume_fraction_primary_phase'
>>> VariableCatalog.VOLUME_FRACTION_SECONDARY_PHASE.name
'volume_fraction_secondary_phase'
>>>
```

Please let me know if any other changes are required.

Thank you.

Reply:
Thanks for the details.